seedlet/assets/sass/blocks/utilities/_style.scss

```
@@ -41,16 +38,13 @@
 	margin-bottom: var(--global--spacing-vertical);
 	/*rtl:ignore*/
 	margin-left: var(--global--spacing-horizontal);
-}
-
-.entry-content > .alignright {
 	@extend %responsive-alignright-mobile;
 	@extend %responsive-alignright;
 }
 
 // Make sure siblings of floated elements are top-aligned when nested
-.entry-content * > .alignleft + *,
-.entry-content * > .alignright + * {
+* > .alignleft + *,
+* > .alignright + * {
```

I think we can replace that `*` selector with the nested block wrapper since this style is specific to nested blocks. Otherwise, it might effect top level blocks as well. 

```suggestion
[class*="inner-container"] > .alignleft + *,
[class*="inner-container"] > .alignright + * {
```

---

Good call, done!